libcore/src/routing/ff_router/ffRouter.cpp

```diff
@@ -398,7 +400,9 @@ int FFRouter::FindExit(Pedestrian * p)
         for(auto & transI : _building->GetRoom(p->GetRoomID())
                                 ->GetSubRoom(p->GetSubRoomID())
                                 ->GetAllTransitions()) {
-            DoorUIDsOfRoom.emplace_back(transI->GetUniqueID());
+            if(!transI->IsClose()) {
```

Comment:
Couldn't we use IsOpen to avoid the negation?

Reply:
Unfortunately no, the door should also be considered if the door is temp_close. Maybe at some point a function which checks for open and temp_close should be introduced.